install.sh

```diff
@@ -897,7 +902,7 @@ case $platform in
     filetype="rpm"
     platform_package="el"
     # For Amazon Linux 2023 and onwards we can use the 'amazon' packages created instead of 'el' packages
-    if (( $platform_version >= 2023 )); then
+    if (( $platform_version >= 2023 || $platform_version == 2 )); then
```

Comment:
I've got some concern about this, it seems like we'll have to come back and update this for each new release of Amazon Linux. Can we get away with just `$platform_version >= 2`?